frontend/public/scss/home-page/home-page-contact-row.scss

```diff
@@ -117,7 +117,10 @@ body.new-design {
               max-width: 100%;
             }
             label {
```

Comment:
Can we hide elements with class `hs-form-required` from the screen reader? This works but it's a little awkward sounding since it reads the label and the required `*` as "email star".